src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -8,10 +8,10 @@
 // distribute, sublicense, and/or sell copies of the Software, and to
 // permit persons to whom the Software is furnished to do so, subject to
 // the following conditions:
-// 
+//
```

Comment:
Quite a bit of trimmed whitespace since I started using the extension that @jnm2 recommended. Not sure I like that much pollution in the PRs ☹️